modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1835,9 +1834,7 @@ private CompletableFuture<Void> handleChangePendingAssignmentEvent(
             // In case of forced assignments we need to remove nodes that are present in the stable set but are missing from the
             // pending set. Such operation removes dead stable nodes from the resulting stable set, which guarantees that we will
```

Comment:
I guess I should change the comment

Reply:
I don't think so, but you can fix it later.

Reply:
I believe that comment should be changed, currently it seems misleading.